### PR TITLE
Happy path so-acceptance tests for /not-arrived

### PR DESCRIPTION
**Pull request by JoeChapman**

### CODE REVIEW

**Comment by sulthan-ahmed** [comment]
I get a failed test on this one 
```
Not Arrived - Happy path: An applicaton can be completed end-to-end:
     An element could not be located on the page using the given search parameters.
```

File: apps/not-arrived/acceptance/features/_happy-path.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+Feature('Not Arrived - Happy path');
+
+const values = {
+  'received': 'yes',
+  'no-letter': 'false',
+  'delivery-date-month': '6',
+  'delivery-date-year': '2017',
```
Comment (sulthan-ahmed) [inline apps/not-arrived/acceptance/features/_happy-path.js]: does this not need a delivery-date-day?

Comment (easternbloc) [inline apps/not-arrived/acceptance/features/_happy-path.js]: Nope but these days should probably be dynamic else they're gonna fail in the future...

**Comment by lennym** [comment]
I feel like for these tests where we possibly expect the user to be taken along a particular journey then it's probably worth actually testing that they are taken on said journey.

All this tests at the moment is that it is *possible* to get from the start to the confirmation page, and not any intermediate functionality.

I'd probably want in this case to at least check that the user is taken through `/not-arrived/same-address`.

**Comment by lennym** [comment]
Note: this can be done with the same code, but listing the expected steps - and omitting any which are not in scope for the test:

```js
I.completeToStep('/not-arrived/same-address', values);
I.completeToStep('/not-arrived/confirmation', values);
```

**Comment by JoeChapman** [comment]
@lennym  can we agree that is out of scope for now? These are quite literally tests with the intention   of testing...
> it is possible to get from the start to the confirmation page

**Comment by lennym** [comment]
Sure, we just need to be aware of the fact that it's not testing very much at all.

**Pull request merged**